test/parallel/test-child-process-silent.js

```diff
@@ -62,7 +62,7 @@ if (process.argv[2] === 'pipe') {
     stdoutData = true;
   });
   let stderrData = false;
-  parent.stdout.on('data', function() {
+  parent.stderr.on('data', function() {
```

Comment:
I am not sure why this was changed?

Reply:
@BridgeAR the original seems wrong as there is already a listener for `stdout` above and this should actually work on `stderr`.